internal/server/utils/bitfield.go

```diff
@@ -0,0 +1,160 @@
+package utils
+
+import (
+	"strconv"
+	"strings"
+
+	diceerrors "github.com/dicedb/dice/internal/errors"
+)
+
+type BitFieldOp struct {
+	Kind   string
+	EType  string
+	EVal   int64
+	Offset int64
+	Value  int64
+}
+
+// parseEncodingAndOffet function parses offset and encoding type for bitfield commands
+// as this part is common to all subcommands
+func parseEncodingAndOffset(args []string) (eType, eVal, offset interface{}, err error) {
```

Comment:
Maybe we can name this better
```suggestion
func parseBitfieldEncodingAndOffset(args []string) (eType, eVal, offset interface{}, err error) {
```

Reply:
Updated in this [commit](https://github.com/DiceDB/dice/commit/b4601acfe13a6de265db09477dc84a3597e2fe26)